src/API.jl

```diff
@@ -487,7 +487,10 @@ function status(ctx::Context, pkgs::Vector{PackageSpec}; mode=PKGMODE_PROJECT)
 end
 
 
-activate() = (Base.ACTIVE_PROJECT[] = nothing)
+function activate()
+    @info("setting active environment to follow working directory")
```

Comment:
This is not correct. If anything it should print the value of `Base.active_project()` after `Base.ACTIVE_PROJECT[]` is set to `nothing`.

Reply:
Huh, I don't know where I got that idea from. I clearly remember reading something that says that, but of course nothing like that exists...

Sorry! You're absolutely correct.